src/main/resources/org/jenkinsci/plugins/docker/workflow/declarative/FolderConfig/config.groovy

```diff
@@ -28,9 +28,7 @@ package org.jenkinsci.plugins.docker.workflow.declarative.FolderConfig
 
 def f = namespace(lib.FormTagLib)
 
-f.section(title:_("Declarative Pipeline (Docker)")) {
```

Comment:
@fqueiruga Is there a way to group related controls together with the new layout, or is the only fix to just delete the section title?

Reply:
I tried to make changes in `folder-plus` as well as `cloudbees-folder`, but it was in the jelly and without any real success

Reply:
I need a bit more context on this one, can please provide some screenshots? 
Did you take a look at https://www.jenkins.io/doc/developer/views/table-to-div-migration, in case it helps? Also pinging @timja

Reply:
@fqueiruga I thought this was related to recent UI changes, but looking again I guess not. It's analogous to https://github.com/jenkinsci/jira-plugin/pull/187. The problem is that using `<f:section>` to group related folder properties contributed by a plugin (e.g. docker-workflow) causes properties contributed by _other_ plugins (e.g. Jira) to show up inside of the `<f:section>` for the first plugin (docker-workflow), which is very confusing. (I think the underlying issue might be that there is already an `<f:section>` surrounding all folder properties and nesting them doesn't always work correctly?)

The fix for the Jira plugin was to just remove the `<f:section>`, which looks fine since there is only one property anyway. In this case there are two properties so removing the grouping completely seems like it isn't ideal, so I just wanted to check if there is a better way to group the elements (because I thought this was related to a recent UI change). I think it isn't related to a recent UI change though now that I looked again, so probably it's easiest to just drop the `<f:section>` and move on.